py3status/py3.py

```diff
@@ -608,10 +609,10 @@ def format_contains(self, format_string, name):
         will fail if the format string contains placeholder formatting
         eg ``'{placeholder:.2f}'``
         """
-
         # We cache things to prevent parsing the format_string more than needed
+        names = [names] if isinstance(names, basestring) else names
         try:
-            return self._format_placeholders_cache[format_string][name]
+            return self._format_placeholders_cache[format_string][str(names)]
```

Comment:
I don't like this.

if names is a `str` with have made it a `list` then a `str` again.

just make life easy
```
if isinstance(names, list):
    key = str(names)
else:
    key = names
    names = [names]

...
```